src/components/ContextMenuView/ChatContextMenuView.tsx

```diff
@@ -34,8 +36,8 @@ const TargetChatContextMenu: BetterSocialContextMenuAction[] = [
   {title: 'Copy', systemIcon: 'square.on.square'}
 ];
 
-const ChatContextMenuView = ({children, contextMenuType, chat}: ChatContextMenuViewProps) => {
-  const {copyMessage, deleteMessage} = useMessageHook(chat);
+const ChatContextMenuView = ({children, contextMenuType, chat, type}: ChatContextMenuViewProps) => {
+  const {copyMessage, deleteMessage} = useMessageHook(chat, type);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are appropriate. The `useMessageHook` function now accepts an additional parameter `type`, which is passed correctly. This change allows for better handling of different chat types.
```diff
- const ChatContextMenuView = ({children, contextMenuType, chat}: ChatContextMenuViewProps) => {
-   const {copyMessage, deleteMessage} = useMessageHook(chat);
+ const ChatContextMenuView = ({children, contextMenuType, chat, type}: ChatContextMenuViewProps) => {
+   const {copyMessage, deleteMessage} = useMessageHook(chat, type);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->